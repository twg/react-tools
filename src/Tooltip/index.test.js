/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import { Tooltip } from './'

describe('Tooltip', () => {
  it('matches snapshot', () => {
    let component = renderer.create(<Tooltip />)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
