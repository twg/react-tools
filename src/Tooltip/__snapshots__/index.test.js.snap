// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip matches snapshot 1`] = `
<div
  className="tooltip"
>
  <span
    className="trigger"
  />
  <div
    className="popover "
  />
</div>
`;
